src/realtps_import/src/client.rs

```diff
@@ -154,6 +157,68 @@ impl Client for SolanaClient {
     }
 }
 
+pub struct StellarClient {
+    client: HorizonHttpClient,
+}
+
+impl StellarClient {
+    pub fn new(url: &str) -> Result<Self> {
+        Ok(Self {
+            client: HorizonHttpClient::new(url)?,
+        })
+    }
+}
+
+#[async_trait]
+impl Client for StellarClient {
+    async fn client_version(&self) -> Result<String> {
+        Ok(stellar_horizon::VERSION.into())
+    }
+    async fn get_latest_block_number(&self) -> Result<u64> {
+        let req = stellar_horizon::api::ledgers::all()
+            .with_limit(1)
+            .with_order(&Order::Descending);
+        let (_headers, mut ledgers) = self.client.request(req).await?;
+        let ledger: Ledger = match ledgers.records.pop() {
+            Some(lg) => lg,
+            None => return Err(anyhow!("request returned empty set of ledgers")),
+        };
+        Ok(ledger.sequence as u64)
+    }
+    async fn get_block(&self, block_number: u64) -> Result<Option<Block>> {
+        if block_number > i32::MAX as u64 {
+            return Err(anyhow!("ledger number out of range"));
+        }
+        let req = stellar_horizon::api::ledgers::single(block_number as i32);
+        let (_headers, ledger) = self.client.request(req).await?;
+        let parent_hash = match ledger.previous_hash {
+            Some(h) => h,
+            None => return Err(anyhow!("missing parent hash")),
+        };
+        Ok(Some(Block {
+            chain: Chain::Stellar,
+            block_number,
+            prev_block_number: if block_number > 0 {
+                Some(block_number - 1)
+            } else {
+                None
+            },
+            timestamp: ledger.closed_at.timestamp() as u64,
+            num_txs: ledger.operation_count as u64,
+            // NB: operation_count corresponds most-closely to what is usually
+            // meant by a "transaction" -- a payment, a trade, etc. Stellar allows
+            // bundling these together into multi-operation "transactions" that
+            // execute atomically under a single fee / signature, so if you prefer
+            // to measure that you can switch the above to the following (though
+            // obviously that reduces the reported number by quite a bit):
+            //
+            // num_txs: ledger.successful_transaction_count
```

Comment:
I think this note is worth shortening this note for a few reasons, because I think the mention of the following two things are not relevant to TPS, or at least are not better measured by the `ledger.successful_transaction_count`:

>single fee

When operations are bundled together the transaction still pays for every operation. The cost of 100 operations submitted as 100 transactions, or 100 operations submitted in a single transaction, are the same. So I don't think the single fee mention here is meaningful to TPS.

>single ... signature

If the goal is to measure the number of signatures verified per block, as is implied here, we'd need a different stat than `ledger.successful_transaction_count`, because transactions can have 1 to 20 signatures. Horizon doesn't provide a metric for how many signatures were verified in a block and it'd be pretty hard to figure that out from the data that Horizon exposes.

```suggestion
            // NB: operation_count corresponds most-closely to what is usually
            // meant by a "transaction" -- a payment, a trade, etc. Stellar allows
            // bundling these together into multi-operation "transactions" that
            // execute atomically.
```

@graydon Wdyt?